data/keycloak/keycloak_export.json

```diff
@@ -0,0 +1,4080 @@
+[ {
+  "id" : "master",
+  "realm" : "master",
```

Comment:
Is this the same as [here](https://github.com/terrestris/shogun-docker/blob/main/shogun-keycloak/init_data/keycloak_export.json)? Maybe this can be loaded from `shogun-docker` repo to keep this fiels in sync